scripts/distro/rke2/distro.sh

```diff
@@ -60,6 +60,7 @@ function rke2_reset() {
 
 function rke2_containerd_restart() {
     rke2_restart
+    sleep 10        # In practice it takes some time for the workloads to end.
```

Comment:
would it be possible to monitor the systemd service?

Reply:
Yes, pushed an update.